tests/test_command_status.py

```diff
@@ -0,0 +1,77 @@
+from file_writer_control.CommandStatus import CommandStatus, CommandState
+import pytest
+
+
+def test_init():
+    job_id = "some job id"
+    command_id = "some command id"
+    under_test1 = CommandStatus(job_id=job_id, command_id=command_id)
+    assert under_test1.job_id == job_id
+    assert under_test1.command_id == command_id
+    assert under_test1.state == CommandState.NO_COMMAND
+
+def test_eq_exception():
+    under_test1 = CommandStatus("job_id", "command_id")
+    with pytest.raises(NotImplementedError):
+        under_test1 == 1
+
+
+def test_eq_success():
+    under_test1 = CommandStatus("job_id", "command_id")
+    under_test2 = CommandStatus("job_id", "command_id")
+    assert under_test1 == under_test2
+
+
+def test_eq_failure_1():
+    under_test1 = CommandStatus("job_id1", "command_id1")
+    under_test2 = CommandStatus("job_id2", "command_id1")
+    assert under_test1 != under_test2
+
+
+def test_eq_failure_2():
+    under_test1 = CommandStatus("job_id1", "command_id1")
+    under_test2 = CommandStatus("job_id1", "command_id2")
+    assert under_test1 != under_test2
```

Comment:
These two tests, failure 1&2, can be combined by using pytest parametrisation. See eg [here](https://docs.pytest.org/en/stable/parametrize.html).

Reply:
A fair suggestion. However, when implementing this I noticed that parametrisation in this case resulted in more code (about 15% more). Hence I suggest that parametrisation not be done in this particular case. Still, a method to keep in mind for the future.

Reply:
I would prefer the longer code to a repeating one. However, if you feel that this is more optimal, sure.